documentation/modules/security/con-configuring-client-quotas.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-kafka-clients.adoc
+
+[id='con-configuring-client-quotas-{context}']
+= Configuring user quotas
+
+[role="_abstract"]
+Configure the `spec` for the `KafkaUser` resource to enforce quotas so that a user does not exceed a configured level of access to Kafka brokers.
```

Comment:
`level of access` for quotas seems strange. Maybe something like `does not overload the Kafka brokers` or something like that would be better.